src/main/resources/de/qabel/desktop/ui/about/about.fxml

```diff
@@ -3,12 +3,21 @@
 <?import javafx.scene.control.*?>
 <?import java.lang.*?>
 <?import javafx.scene.layout.*?>
+<?import javafx.scene.image.ImageView?>
+<?import javafx.scene.image.Image?>
 
-
-<ScrollPane fx:controller="de.qabel.desktop.ui.about.AboutController" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
+<ScrollPane fx:controller="de.qabel.desktop.ui.about.AboutController" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1" >
     <content>
-        <VBox fx:id="linkContainer">
+        <VBox fx:id="linkContainer"  styleClass="linkcontainer-about">
             <children>
+                <ImageView styleClass="img-logo-about" pickOnBounds="true" preserveRatio="true" >
+                    <image>
+                        <Image url="@/img/logo-with-name.png" />
+                    </image>
+                </ImageView>
+                <Label text="%desktopAppVersion" />
```

Comment:
If you need to show a version, take a look at the DesktopClient::checkVersion, it uses the current version. Maybe we should integrate it to the DI-Container. But that's probably out of scope for this issue